cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -427,12 +425,16 @@
               Verify that the CWT is a valid CBOR object.
             </t>
             <t>
-              If the object begins with the CWT CBOR tag, remove it
-	      and verify that one of the COSE CBOR tags follows it.
-	    </t>
-	    <t>
-	      If the object is tagged with one of the COSE CBOR tags, remove it
-	      and verify that it corresponds to the structure of the following COSE object.
+              If the object begins with the CWT CBOR tag, remove it and
+              continue verification.
```

Comment:
I thought we had agreed that that if the CWT CBOR tag is present, it must be followed by a COSE CBOR tag.  This change seems to allow the COSE CBOR tag to not be present.  Was that intentional?